deal.II-toolchain/platforms/supported/debian10.platform

```diff
@@ -0,0 +1,23 @@
+# debian 10
+#
+# This build script assumes that you have several packages already
+# installed via debian's apt-get using the following commands:
+#
+# > sudo apt-get install build-essential automake autoconf gfortran \
+#   openmpi-bin openmpi-common libopenmpi-dev cmake subversion git \
+#   libblas-dev liblapack-dev libblas3 liblapack3 splint tcl tcl-dev \
+#   environment-modules libsuitesparse-dev libtool libboost-all-dev \
+#   qt4-dev-tools
+#
+# Then reboot and run candi again.
+##
+
+## on debian 10 the candi installed parmetis 4.0.3 is not recognized correctly
+# for trilinos 12-10-1. We force to assume parmetis version 4.0.3.
+TRILINOS_PARMETIS_CONFOPTS="\
```

Comment:
can you remove these few lines and comments, please? I don't think this is necessary.

Reply:
I agree, this should not be necessary.
These lines are also present in the platforms: centos7, debian9 and rhel7. I also removed the lines in these files.